tests/custom-actions/test_action_block.c

```diff
@@ -907,35 +1013,96 @@ Test(exec_action_block, gen)
     attr4->attribute_tag = INTEGER;
     val4.int_val = 0;
     attr4->attribute_value = val4;
- 
-    args1 = (attribute_t**) malloc(sizeof(attribute_t*) * 3);
+
+    args1 = (attribute_t **)malloc(sizeof(attribute_t *) * 3);
     args1[0] = attr1;
     args1[1] = attr2;
     args1[2] = attr4;
-    
-    args2 = (attribute_t**) malloc(sizeof(attribute_t*) * 3);
+
+    args2 = (attribute_t **)malloc(sizeof(attribute_t *) * 3);
     args2[0] = attr1;
     args2[1] = attr3;
     args2[2] = attr4;
- 
-    act1 = action_block_new(GEN, 3, args1);
-    act2 = action_block_new(GEN, 2, args1);
-    act3 = action_block_new(GEN, 3, args2);
-    
+
+    action_params1 = (action_param_t *)malloc(sizeof(action_param_t));
+    action_params2 = (action_param_t *)malloc(sizeof(action_param_t));
```

Comment:
```suggestion
    action_params2 = (action_param_t*)malloc(sizeof(action_param_t));
```
Spaces for lines 1027 and 1028 (see [here](https://github.com/uchicago-cs/chiventure/pull/1373/files#r872989077))